### ISSUE TITLE
what is prv_cut in ancombc function really mean ?

### ISSUE THREAD

**Issue by Hesham999666** [FrederickHuangLin/ANCOMBC#101]
This what is written in Arguments explanation  

a numerical fraction between 0 and 1. Taxa with prevalences less than prv_cut will be excluded in the analysis. Default is 0.10.
Is the taxa prevalences for total samples or for each samples alone ?

**Comment by FrederickHuangLin** [comment]
Hi @Hesham999666,

I have updated the description on `prv_cut` in the new release of `ANCOMBC` package:

> a numerical fraction between 0 and 1. Taxa with prevalences less than `prv_cut` will be excluded in the analysis. For instance, suppose there are 100 samples, if a taxon has nonzero counts presented in less than 10 samples, it will not be further analyzed. Default is 0.10.

Also, I am pleased to share with you that the major update for `ANCOMBC` package has been completed. Here are some highlighted new features:

- Add `ancombc2` function, which supports pairwise test and trend test for both cross-sectional and repeated measurements data
- Add sensitivity analysis for pseudo-count addition
- Fully support the `(Tree)SummarizedExperiment` class
- A more user-friendly output layout

I just pushed the changes to the Bioconductor branches. It might take a few business days for the package to become available (but the `devel` version is ready here!).

Best,
Huang
